turbinia/turbiniactl.py

```diff
@@ -272,6 +274,13 @@ def main():
   subparsers.add_parser('server', help='Run Turbinia Server')
 
   args = parser.parse_args()
+  if args.config:
+    try:
+      with open(config.configSource(), "r") as f:
```

Comment:
```suggestion
      with open(config.configSource, "r") as f:
```
This is just a string.